### PR TITLE
LPS-184157 | Test Fix | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-184157](https://issues.liferay.com/browse/LPS-184157)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
ci:test:workflow-metrics

**Comment by patriciajeanperez** [comment]
Hey @victorhcunha, can you give a short description of the issue and how this will fix it? Thanks!

**Comment by victorhcunha** [comment]
Hi @patriciajeanperez !
I will leave some comment
The pr was in the draft while I focused on other activities

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase [hunk 1]
```diff
@@ -966,10 +966,12 @@ definition {
...
 	@priority = 3
 	test SLAProcessingTimeFormatIsFollowingSystemLanguage {
-		property ci.retries.disabled = "true";
-
 		var slaName = "SLAName";
 
+		WorkflowMetrics.setSLADefinitionsJobInterval(
+			definitionsJobInterval = 1,
+			slaJobInterval = 1);
+
```
Comment (victorhcunha) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase:974]: Like in https://github.com/liferay-bpm-qa/liferay-portal/pull/839
This step is for metrics recalculation. To ensure that the data has been read before asserting at the end of the test.

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase [hunk 2]
```diff
@@ -1002,6 +1004,10 @@ definition {
 
 		var getInstanceId = selenium.getText("WorkflowAllItems#WORKFLOW_INSTANCE_ID");
 
+		while ((IsElementNotPresent(key_status = "check", locator1 = "WorkflowAllItems#DETAIL_SLA_STATUS")) && (maxIterations = "50")) {
+			Refresh();
+		}
+
```
Comment (victorhcunha) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase:1010]: This is step is to ensure that the poshi will wait for the status update (after SLADefinitionsJobInterval) and don't  try to click before this

**Comment by victorhcunha** [comment]
This test is affected by bug [LPS-186021](https://issues.liferay.com/browse/LPS-186021)

**Comment by patriciajeanperez** [comment]
ci:test:workflow-metrics

**Comment by patriciajeanperez** [comment]
@davisaints Hi Davi! Can you review this PR?

**Comment by davisaints** [comment]
Sure!

**Comment by davisaints** [comment]
Sure!

**Comment by davisaints** [comment]
ci:test:workflow-metrics

**Comment by davisaints** [comment]
LGTM!

**Comment by davisaints** [comment]
ci:forward

**Comment by davisaints** [comment]
ci:stop

**Comment by davisaints** [comment]
ci:forward:force

**Pull request closed without merging**